sshuttle/client.py

```diff
@@ -391,6 +396,10 @@ def expire_connections(now, mux):
     debug3('Remaining UDP channels: %d\n' % len(udp_by_src))
 
 def check_connections_allowed(mux):
+    # if alwaysConnected mode is on, all connections are allowed. we don't need to check them
+    if _always_connected == ALWAYS_CONNECTED_ON:
```

Comment:
if you do this check, won't any change to the Firewall be ignored for pre-existing connections?  I.e. you connect to server1:port1, and you remove the ability in firewall to connect... the existing connections should drop.

Reply:
you're right, this is leftover from when I thought we were skipping firewall rules